planemo/options.py

```diff
@@ -575,6 +575,16 @@ def conda_global_option():
     )
 
 
+def noverify_option():
+    return planemo_option(
+        "--noverify",
```

Comment:
The rest of the commands separate words - this should be no_verify right?

Reply:
yes of course. I will modify or we will jut drop this: https://github.com/galaxyproject/planemo/issues/981#issuecomment-617308493